Sources/StreamVideoSwiftUI/CallingViews/iOS13/PreJoiningView_iOS13.swift

```diff
@@ -11,7 +11,7 @@ struct LobbyView_iOS13: View {
     
     @ObservedObject var callViewModel: CallViewModel
     @BackportStateObject var viewModel: LobbyViewModel
-    @BackportStateObject var microphoneChecker = MicrophoneChecker()
+    @Injected(\.microphoneChecker) var microphoneChecker: MicrophoneChecker
```

Comment:
hm, did you test on iOS 13 to make sure it still works there?

Reply:
It has actually become quite difficult to test iOS 13 on Xcode 14/15 (i haven't tried it in a while). The code structure is now as it used to be but it may take a while to be able to test it.